samples/snippets/requirements.in

```diff
@@ -0,0 +1,15 @@
+google-api-core>=1.30.0
+google-auth<3.0.0dev,>=1.25.0
+google-cloud-bigquery<3.0.0dev,>=2.25.2
+sqlalchemy<1.5.0dev,>=1.2.0
+future
+alembic
+pytz
+shapely
+GeoAlchemy2
+packaging
+alembic
+GeoAlchemy2
+shapely
+packaging
+pytz
```

Comment:
We prefer not to be generating these files:  we use the Whitesource Renovate bot to keep the hand-rolled versions up-to-date.

Reply:
Can you trigger it for this repo? Part of the problem here is that the original `samples/snippets/requirements.txt` requires [Fiona](https://pypi.org/project/Fiona/)==1.8.20 and this version does not include support for python 3.10, though I also don't see why Fiona is in there to begin with.

Reply:
If you want I can just commit the `requirements{.test,}.txt` files and remove `requirements{.test,}.in` files.

Reply:
> If you want I can just commit the requirements{.test,}.txt files and remove requirements{.test,}.in files

Please do.

> I also don't see why Fiona is in there to begin with.

I believe it came in as a transitive dependency of geopandas. Though, I don't see geopandas in use here: https://github.com/googleapis/python-bigquery-sqlalchemy/blob/main/samples/snippets/geography.py so perhaps leftover from an old sample.

Reply:
Done